vengine/vengine/application/Input.hpp

```diff
@@ -167,4 +168,5 @@ class Input
 	static inline const int getMouseDeltaX() { return Input::deltaCursorX; }
 	static inline const int getMouseDeltaY() { return Input::deltaCursorY; }
 	static inline const int getScrollWheelDelta() { return Input::deltaScrollWheel; }
+	static const glm::ivec2 getMouseUITranslated();
```

Comment:
Please change to glm::vec2 for the return type. This is to accurately handle the mouse cursor on high resolution displays.